doc/examples/Finite_State_Projection/utils.py

```diff
@@ -0,0 +1,123 @@
+import amici
+import amici.plotting
+
+import importlib
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import sys
+
+
+def plot_AMICI(sbml_dir: str,
+               t: np.ndarray,
+               r_max: int,
+               p_max: int):
+
+    n_t = len(t)
+    # compile and Simulate model
+    simulation, model = compile_and_simulate(sbml_dir,
+                                             t,
+                                             r_max,
+                                             p_max)
+
+    def get_obs_idx_by_id(obs_id: str):
+        for idx, obs in enumerate(model.getObservableIds()):
+            if obs == obs_id:
+                return idx
+        raise IndexError(f'No observable with id {obs_id}')
```

Comment:
```suggestion
        try:
            return model.getObservableIds().index(obs_id)
        except ValueError:
            raise IndexError(f'No observable with id {obs_id}')
```

Same logic but may be faster/more conventional